staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/meta.go

```diff
@@ -171,12 +172,12 @@ func (meta *ObjectMeta) SetOwnerReferences(references []OwnerReference) {
 func (meta *ObjectMeta) GetClusterName() string            { return meta.ClusterName }
 func (meta *ObjectMeta) SetClusterName(clusterName string) { meta.ClusterName = clusterName }
 
-func (meta *ObjectMeta) GetLastApplied() map[string]string {
+func (meta *ObjectMeta) GetLastApplied() map[string]runtime.RawExtension {
 	return meta.LastApplied
 }
 
-func (meta *ObjectMeta) SetLastApplied(lastApplied map[string]string) {
-	meta.LastApplied = make(map[string]string, len(lastApplied))
+func (meta *ObjectMeta) SetLastApplied(lastApplied map[string]runtime.RawExtension) {
+	meta.LastApplied = make(map[string]runtime.RawExtension, len(lastApplied))
```

Comment:
Wouldn't it be safer to copy in the get function instead of or in addition to here?

alternatively, we could provide get/set/delete functions that took the key? Plus a list, I guess. Hm.

Reply:
How efficient/safe is it to copy a RawExtension?